src/components/new-asset-form/styles.scss

```diff
@@ -16,20 +10,36 @@
     }
 }
 
-#submit-new-property-button {
-    margin-right: 20px;
-}
-
 .patch {
     display: flex;
 }
 
-#patch-add-link {
-    font-size: 16px !important;
-}
-
 .patch-remove-link {
     margin-left: 10px;
     margin-top: 0px;
     font-size: 16px !important;
+}
+
+#new-property-form {
+
+    input,
+    select {
+        width: 500px;
+    }
+}
+
+#submit-new-property-button {
+    margin-right: 20px;
+}
+
+#patch-add-link {
+    font-size: 16px !important;
+}
+
+#property-patches-container {
+    margin-top: 0;
+}
+
+#new-property-disclaimer {
+    margin-top: 28.5px;
```

Comment:
Is there a reason you've used id's for selectors? Using classes is the normal convention.

Reply:
Using a class would make sense if I were creating reusable styling that could be re-used for multiple elements. If the styling you're creating is specific to a single item I think ID is preferable.

Reply:
Im not sure. Ive rarely seen anyone use ids when writing css